smokes/e2e/force.scenarios.coffee

```diff
@@ -1,16 +1,35 @@
-describe 'force', () ->
-    it 'should create a build', () ->
-        browser.get('#/builders/1')
-        lastbuild = element.all(By.css('span.badge-status.results_SUCCESS')).count()
-        browser.get('#/builders/1/force/force')
-        browser.waitForAngular()
-        element(By.buttonText('Start Build')).click()
-        browser.get('#/builders/1')
-        successBuildIncrease =  ->
-            lastbuild.then (lastbuild)->
-                element.all(By.css('span.badge-status.results_SUCCESS'))
-                .count().then (nextbuild)->
-                    return +nextbuild == +lastbuild + 1
-        browser.wait(successBuildIncrease, 20000)
-        browser.get('#/waterfall')
-        expect(element.all(By.css('rect.success')).count()).toBeGreaterThan(0)
+# coffee script 
+# test goal: checks the capability to define a reason and to cancel/start the build
+
+forceSchedulerPage = require('./reason_force.coffee')
+waterFallPage = require('./waterfall.coffee')
+
+forcePage = null;
+waterfallPage = null;
+
+describe('', () ->
+    beforeEach(() ->
+        forcePage =  new forceSchedulerPage();
+        waterfallPage = new waterFallPage();
+        forcePage.goToDefaultURL();
+    )
+
+
+
+    describe 'force', () ->
+        it 'should create a build', () ->
+        
+            lastbuild = element.all(By.css('span.badge-status.results_SUCCESS')).count();
+            forcePage.goToURL('#/builders/1/');
+            forcePage.goToURL('#/builders/1/force/force');
```

Comment:
goToURL is better. but actually I think it is better to hardcode the URL in the page classes.
So that would be

    forcePage.go()

without the need to first go to the builders page